common/src/dodeca.rs

```diff
@@ -125,6 +125,8 @@ impl Vertex {
         })
     }
 
+    /// Returns the transform which places this cube in its canonical position
+    /// relative to the node it's associated with.
```

Comment:
This is unclear. "Canonical position" is not a term we define or use anywhere, and this does not express that this is actually a very strange matrix in that it translates from euclidean coordinates to hyperboloid model coordinates.

Maybe "Transform from euclidean cube coordinates to the hyperbolic space relative to the cube's canonical node"?